backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -63,7 +63,7 @@
           end
         end
 
-        # Click "ok" on confirmation dialog
+        expect(page).to have_content("YOUR ORDER IS EMPTY") # wait for page refresh
```

Comment:
Just for my own edification -- why does this fix things?  From you comment it sounds like you're adding this because this line makes the specs wait until the page refreshes?  Does the `not_to have_content` behave differently? (doesn't wait for the refresh?)